tests/commoncode/test_fileset.py

```diff
@@ -30,14 +30,16 @@
 import commoncode.testcase
 from commoncode import fileset
 
+import pytest
+pytestmark = pytest.mark.scanpy3 #NOQA
 
 class FilesetTest(commoncode.testcase.FileBasedTesting):
     test_data_dir = os.path.join(os.path.dirname(__file__), 'data')
 
     def test_load(self):
         irf = self.get_test_loc('fileset/scancodeignore.lst')
         result = fileset.load(irf)
-        assert ['/foo/*', '!/foobar/*', 'bar/*', '#comment'] == result
+        assert [b'/foo/*', b'!/foobar/*', b'bar/*', b'#comment'] == result
```

Comment:
How could this possibly work on Python 3?

Reply:
console says bytes like object required not str

Reply:
So may be the file should be opened in binary mode only on Python 2?